translations/en.json

```diff
@@ -1,7 +1,7 @@
 {
-  "terraApplication.unsavedChangesPrompt.title": "There are unsaved changes.",
-  "terraApplication.unsavedChangesPrompt.singlePromptMessage": "There are unsaved changes made to {promptDescription}. Changes will be lost if you don't save them. How do you want to proceed?",
-  "terraApplication.unsavedChangesPrompt.multiplePromptMessageIntro": "There are unsaved changes made to the following:",
+  "terraApplication.unsavedChangesPrompt.title": "Unsaved changes",
```

Comment:
@emilyrohrbough - One of the late changes the UX group decided was to update the titles as they were posted on the Design Standards from sentence case to title case (i.e. book titles, article titles, etc) capitalization -- so this should read "Unsaved **C**hanges" instead of "Unsaved **c**hanges".

We should at least change it for English and then do a followup on the original jira with Translations before this is final to explain the capitalization change and see how the other languages would need to change to use capital letters as well.

Reply:
I checked with Brett Gray on the translation team. He confirmed english translations will needed updated to the title case, however, all other locales can remain as is. I will make those updates & regenerate screenshots.

Reply:
updated: https://github.com/cerner/terra-application/pull/27/commits/6cefd172d405a8e441d4c5cd2dca5435d5addf3b